frontend/src/metabase/admin/permissions/components/tree/TreeNode.styled.jsx

```diff
@@ -0,0 +1,60 @@
+import styled, { css } from "styled-components";
+import colors from "metabase/lib/colors";
+import Icon from "metabase/components/Icon";
+
+export const TreeNodeRoot = styled.li`
+  display: flex;
+  align-items: center;
+  color: ${props =>
+    props.isSelected ? colors["white"] : colors["text-medium"]};
+  background-color: ${props =>
+    props.isSelected ? colors["accent7"] : "unset"};
+  padding-left: ${props => props.depth + 1}rem;
+  padding-right: 0.5rem;
+  cursor: pointer;
+  font-weight: 700;
+
+  &:hover {
+    background-color: ${props =>
+      props.isSelected ? colors["accent7"] : `#e7e6f1`};
```

Comment:
should we put this color somewhere? fwiw I've also seen code like `lighten(colors("accent7"))` that might give you a similar color? not sure.

Reply:
Thanks for the suggestion, I changed that